pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_simple/data_generation.py

```diff
@@ -0,0 +1,229 @@
+#############################################################################
+# Copyright (C) 2020-2022 German Aerospace Center (DLR-SC)
+#
+# Authors: Agatha Schmidt, Henrik Zunker
+#
+# Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#############################################################################
+from memilio.simulation import UncertainContactMatrix, ContactMatrix, Damping
+from memilio.simulation.secir import Model, simulate, AgeGroup, Index_InfectionState, Simulation, interpolate_simulation_result
+from memilio.simulation.secir import InfectionState as State
```

Comment:
```suggestion
from memilio.simulation.secir import InfectionState
```

Please do not name it State ( even if this comes from the other example )